src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -238,16 +236,13 @@ public static void connectPortBus(EDIFCell topCell, EDIFCellInst src, EDIFCellIn
 			EDIFNet net = null;
 			net = new EDIFNet("conn_net_" + EDIFTools.makeNameEDIFCompatible(outputPortName), topCell);
 
-			EDIFPortInst outputPortInst = srcPortMap.get(outputPortName);
-			EDIFPortInst inputPortInst = snkPortMap.get(inputPortName);
+			EDIFPortInst outputPortInst = src.getPortInst(outputPortName);
+			EDIFPortInst inputPortInst = snk.getPortInst(inputPortName);
 			
 			@SuppressWarnings("unused")
 			EDIFPortInst outputPort = new EDIFPortInst(outputPortInst.getPort(),net,outputPortInst.getIndex(),src);
 			@SuppressWarnings("unused")
 			EDIFPortInst inputPort = new EDIFPortInst(inputPortInst.getPort(),net,inputPortInst.getIndex(),snk);
```

Comment:
Why is this necessary?

Reply:
This code was part of a debug demo, just trying not to break it.